doc/modules/preprocessing.rst

```diff
@@ -474,11 +474,11 @@ then the shape of the transformed input is
     Imputer(add_indicator_features=True, axis=0, copy=True, missing_values='NaN',
         strategy='mean', verbose=0)
     >>> print(imp_with_in.transform(X))                           # doctest: +ELLIPSIS
-    [[ 4.          2.          1.          0.        ]
-     [ 6.          3.66666667  0.          1.        ]
+    [[ 4.          2.          0.          0.        ]
```

Comment:
Clearly, you were right to ask questions because you've still missed the point. Have you *used* scikit-learn's transformers? Do you understand their API design? Do you understand what this `add_indicator_features` is for?

Yes, we made an error in reviewing this PR, but not one as glaring as to get this wrong!